core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -3621,7 +3621,9 @@ protected function treeView()
 		$table = $this->strTable;
 		$treeClass = 'tl_tree';
 
-		if (($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['mode'] ?? null) == self::MODE_TREE_EXTENDED)
+		$blnTreeModeExtended = ($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['mode'] ?? null) == self::MODE_TREE_EXTENDED;
```

Comment:
Just following the constant name here:

```suggestion
		$blnModeTreeExtended = ($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['mode'] ?? null) == self::MODE_TREE_EXTENDED;
```

Reply:
Done in bc73ce58f1144d6bf38e97de1aec199bab21e23f :)